### DISCUSSION TITLE
Transferring ownership to another Tesla account

### DISCUSSION THREAD

**Discussion by kocsenc** [teslamate-org/teslamate#1119] [Q&A]
I've had Teslamate collect data for a couple years. For _reasons_, I need to move the ownership of the car to another Tesla Account - curious what's involved so I can have Teslamate continue to log information from the new account to the same vehicle @adriankumpf. 

I'm fairly familiar with SQL and programming so a "hacky" approach is valid.

**Comment by kocsenc** [comment]
I ran into #706 , which outlines removing a vehicle. Would it be possible to maybe change the id of the "new car" on the "new Tesla Account" to the older one?

**Comment by kocsenc** [comment]
Hey @adriankumpf do you have any suggestions for this? What I'm going to try next is bring up a new instance with the new car and see what I can do to match up the data.

**Reply by adriankumpf** [reply]
I would suggest the following: 

1. Move the ownership of the car to the new Tesla Account 
2. Log out from TeslaMate: There is no built-in functionality for this. You would have to delete the tokens from the `tokens` table.
3. Sign in with the new Tesla Account
